runtime/v2/vm/src/main/java/com/walmartlabs/concord/svm/ExecutionListener.java

```diff
@@ -35,12 +35,19 @@ default Result beforeCommand(Runtime runtime, VM vm, State state, ThreadId threa
     }
 
     /**
-     * Called after the last command in the stack was executed.
+     * Called after the command in the stack was executed.
      */
     default Result afterCommand(Runtime runtime, VM vm, State state, ThreadId threadId, Command cmd) {
         return Result.CONTINUE;
     }
 
+    /**
+     *  Called after the command execution ended with exception/error.
+     */
+    default Result afterCommandWithError(Runtime runtime, VM vm, State state, ThreadId threadId, Command cmd, Exception e) {
```

Comment:
`onCommandError` perhaps? It happens right in the exception handler, `on` feels appropriate to me.